examples/step-61/doc/intro.dox

```diff
@@ -139,186 +139,333 @@ that is only defined in cell interiors. Consequently,
+  indices. Which it is should be obvious from the context. In practice,
+  of course, whenever we do local integration on a cell, we only ever
+  consider cell-local indices, and the translation to global indices
+  is done when copying the local contributions into the global matrices.
+  This is first shown in step-3 and step-4. In the current program,
+  this will be facilitated by calling
+  ConstraintMatrix::distribute_local_to_global().
+
+A linear system of course also needs a right hand side. There is no difficulty
+associated with computing the right hand side here other than the fact
+that we only need to use the cell-interior part $\varphi_i^\circ$ for
+each shape function $\varphi_i$.
+
 
 <h3> Post-processing and $L_2$-errors </h3>
 
-After we have calculated the numerical pressure $p$, 
-we use discrete weak gradients of $p$ to calculate the velocity on each element.
+The discussions in the previous sections have given us a linear
+system that we can solve for the numerical pressure $p_h$. We can use
+this to compute an approximation to the variable $\mathbf u = -{\mathbf K}\nabla p$
+that corresponds to the velocity with which the medium flows in a porous
+medium if this is the model we are trying to solve. This kind of
+step -- computing a derived quantity from the solution of the discrete
+problem -- is typically called "post-processing".
 
-On each element the gradient of the numerical pressure $\nabla p$ can be
-approximated by discrete weak gradients  $ \nabla_{w,d}\phi_i$, so
+Here, instead of using the exact gradient of $p_h$, let us instead
+use the discrete weak gradient of $p_h$ to calculate the velocity on each element.
+As discussed above,
+on each element the gradient of the numerical pressure $\nabla p$ can be
+approximated by discrete weak gradients  $ \nabla_{w,d}\phi_i$:
 @f{equation*}
-\nabla_{w,d} p_h = \sum_{i} a_i \nabla_{w,d}\phi_i.
+\nabla_{w,d} p_h 
+= \nabla_{w,d} \left(\sum_{i} P_i \phi_i\right) 
+= \sum_{i} P_i \nabla_{w,d}\phi_i.
 @f}
 
-The numerical velocity $ \mathbf{u}_h = -\mathbf{K} \nabla_{w,d}p_h$ can be written as
-@f{equation*}
-\mathbf{u}_h = -\mathbf{K} \nabla_{w,d} p = 
--\sum_{i} \sum_{j} a_ic_{ij}\mathbf{K}\mathbf{w}_j,
+On cell $K$,
+the numerical velocity $ \mathbf{u}_h = -\mathbf{K} \nabla_{w,d}p_h$ can be written as
+@f{align*}{
+\mathbf{u}_h 
+&= -\mathbf{K} \nabla_{w,d} p = 
+-\mathbf{K}\sum_{i} \sum_{j} P_i C^K_{ij}\mathbf{v}_j
+&= -\nabla_{w,d} p = 
+-\mathbf{K}\sum_{i} \sum_{j} P_i C^K_{ij}\mathbf{K} \mathbf{v}_j,
 @f}
-where $c_{ij}$ is the coefficient of Gram matrix, 
-$\mathbf{w}_j$ is the basis function of the $RT$ space. 
-$\mathbf{K} \mathbf{w}_j$ may not be in the $RT$ space. 
-So we need $L_2$-projection to project it back to the $RT$ space. 
+where $C^K$ is the expansion matrix from above, and
+$\mathbf{v}_j$ is the basis function of the $RT$ space on a cell.
+
+Unfortunately, $\mathbf{K} \mathbf{v}_j$ may not be in the $RT$ space. 
```

Comment:
It stays in the RT space if `K` is a constant matrix, right?

Reply:
Only if it is a constant multiple of the identity matrix. I'll add this.

Reply:
:+1: